src/components/views/dialogs/CreateRoomDialog.tsx

```diff
@@ -215,6 +217,10 @@ export default class CreateRoomDialog extends React.Component<IProps, IState> {
         return result;
     };
 
+    private onIsPublicChange = (isPublic: boolean): void => {
+        this.setState({ isPublic });
```

Comment:
for reference: I was confused about what this `isPublic` state was doing. The answer is: nothing. It appears to have been introduced by https://github.com/matrix-org/matrix-react-sdk/pull/5698, probably due to an incorrect merge resolution due to a conflict with https://github.com/matrix-org/matrix-react-sdk/pull/6212 (which removed it).

Reply:
That was also my conclusion. So, I decided to give it a purpose.